src/module-1/card/stylesheets/card.css

```
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300&display=swap');
+*{
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+.card{
+  height: 486px;
+  max-width: 400px;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.135216);
```

```suggestion
  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.14);
```  
feel free to round values with 3+ digits after dot

---

[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)